core/domain/platform_parameter_domain_test.py

```diff
@@ -0,0 +1,86 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the config variable registry."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.domain import platform_parameter_domain as parameter_domain
+from core.tests import test_utils
+import utils
+
+
+class PlatformParameterChangeTests(test_utils.GenericTestBase):
+    VALID_CMD_NAME = (
```

Comment:
This seems odd -- it doesn't make things particularly generalizable. Perhaps you could have CMD_EDIT_RULES = ... and then refer to that elsewhere.

Reply:
Done, changed the variable naming to `CMD_EDIT_RULES`.